tools/ci_build/github/azure-pipelines/templates/java-api-artifacts-package-and-publish-steps-windows.yml

```diff
@@ -9,15 +9,14 @@ steps:
     - task: CmdLine@2
       inputs:
         script: |
-          echo "Java version"
-          java --version
+          @echo on
           set NATIVE_FOLDER=$(Build.BinariesDirectory)\${{parameters.artifactName}}\stage\ai\onnxruntime\native\win-x64
           mkdir %NATIVE_FOLDER%
           echo "Directories created"
           copy .\java\build\libs\*.jar $(Build.BinariesDirectory)\${{parameters.artifactName}}
           pushd $(Build.BinariesDirectory)\${{parameters.artifactName}}
-          jar xf onnxruntime-${{parameters.version}}.jar META-INF\maven\ai.onnxruntime\onnxruntime\pom.xml
-          move META-INF\maven\ai.onnxruntime\onnxruntime\pom.xml onnxruntime-${{parameters.version}}.pom
+          jar xf onnxruntime-${{parameters.version}}.jar META-INF\maven\com.microsoft.onnxruntime\onnxruntime\pom.xml
+          move META-INF\maven\com.microsoft.onnxruntime\onnxruntime\pom.xml onnxruntime-${{parameters.version}}.pom
```

Comment:
Note if the artifactID has changed to add GPU, then the pom file will be in `com.microsoft.onnxruntime/onnxruntime_gpu/`.

Reply:
This is the interesting bit. When I changed it to _gpu it broke so I had to reverrt. artifactID does not change this path.

---
In reply to: [436856909](https://github.com/microsoft/onnxruntime/pull/4159#discussion_r436856909) [](ancestors = 436856909)

Reply:
See the note I put on line 107, that's because I'm not being specific enough in where I put it, because most of the time project.group == groupId and project.name == artifactId.